draft-ietf-quic-transport.md

```diff
@@ -1352,6 +1358,10 @@ receive times relative to the beginning of the connection.
 ~~~
 {: #timestamp-format title="Timestamp Section"}
 
+The timestamp section is time-ordered, so the 1 byte Delta Largest
+Acknowledged is always relative to the Largest Acknowleged, but the Time
+Since Previous Timestamp fields are relative.
```

Comment:
I would suggest moving this to below the descriptions of the fields. Suggested rewording: "The timestamp section lists packet receipt timestamps ordered by timestamp." I think you can drop the rest of this paragraph since it's redundant with text in the descriptions.